core/services/vrf/listener_v2.go

```diff
@@ -817,9 +818,9 @@ func (lsn *listenerV2) processRequestsPerSub(
 				maxLinkString := p.maxLink.String()
 				requestID := common.BytesToHash(p.req.req.RequestId.Bytes())
 				coordinatorAddress := lsn.coordinator.Address()
-				ethTX, err = lsn.txm.CreateEthTransaction(txmgr.NewTx{
-					FromAddress:    fromAddress,
-					ToAddress:      lsn.coordinator.Address(),
+				transaction, err = lsn.txm.CreateEthTransaction(txmgr.NewTx[*evmtypes.Address]{
+					FromAddress:    evmtypes.NewAddress(fromAddress),
```

Comment:
just a thought: one day it might be nice to use evmtypes.Address across the entire codebase (then we have less conversions happening)

Reply:
We'd still have places that need gethcommon.Address type, to talk to EVM RPC nodes.
But I agree, once we convert all txm, client, head tracker, logpoller to generics, then the only parts that need gethcommon.Address will be the limited ones talking to EVM RPCs. Then we could make this code far more simpler!